compute/cloud-client/src/test/java/compute/InstanceTemplatesIT.java

```diff
@@ -69,11 +71,21 @@ public static void setup() throws IOException, ExecutionException, InterruptedEx
     MACHINE_NAME_2 = "my-new-test-instance" + UUID.randomUUID();
     MACHINE_NAME_3 = "my-new-test-instance" + UUID.randomUUID();
     TEMPLATE_NAME_WITH_DISK = "my-new-test-instance" + UUID.randomUUID();
+    TEMPLATE_NAME_FROM_INSTANCE = "my-new-test-instance" + UUID.randomUUID();
+    TEMPLATE_NAME_WITH_SUBNET = "my-new-test-instance" + UUID.randomUUID();
```

Comment:
Thanks for the feedback, Les. 
I've made the following changes:
1. Re-used same UUID for similar operations
2. The prefix is variable depending upon the operation performed. 

This would allow to group operations related to a specific UUID and would also make deletion simpler by looking for the invariable part in the prefix.
Please lmk your thoughts!